### PR TITLE
updated combinatinalAnalysis.js

### PR DESCRIPTION
**Pull request by sal2701**
Trimmed the whole list and used "comma" to split the inputs

Screenshots:
trim_1
trim_2
trim_3

### CODE REVIEW

**Comment by sal2701** [comment]
@satu0king please review, this in reference to #318

File: public/js/combinationalAnalysis.js [hunk 1]
```diff
@@ -10,17 +10,19 @@ createCombinationalAnalysisPrompt=function(scope=globalScope){
...
       buttons: [
         {
           text: "Next",
           click: function() {
             // //console.log($("#inputNameList"),$("#inputNameList").val(),$("#inputNameList").html());
-            var inputList=$("#inputNameList").val().split(' ');
-            var outputList=$("#outputNameList").val().split(' ');
+            var inputList=$("#inputNameList").val().split(',');
+            var outputList=$("#outputNameList").val().split(',');
+            inputList = inputList.map(function (x){return x.trim();});
```
[the remark this answers is not in the document]
Comment (sal2701) [reply public/js/combinationalAnalysis.js]: Changed it

**Comment by satu0king** [comment]
The circuit is not generating due to some internal error. Looks like the quin-mcluskey implementation cannot handle spaces. Is it working for you?
image

**Comment by sal2701** [comment]
It is not working even for me, spaces are causing the problem like you said

**Comment by satu0king** [comment]
> @satu0king It is not working even for me, spaces are causing the problem like you said

We have to take a deeper look or consider a work around.

**Pull request closed without merging**